lesson-13/js/main.js

```diff
@@ -0,0 +1,65 @@
+function loadSlider() {
+  $(".slick__wrapper").slick({
+    slidesToShow: 4,
+    slidesToScroll: 4,
+    infinite: true,
+    arrows: true,
+  });
+}
+
+const homesInfoPanel = document.querySelector(".favorite-wrapper");
+
+const dataArr = ((data) => {
+  data.forEach((item) => { 
+  const hotelsItem = document.createElement("div");
+ hotelsItem.classList.add("guests-Homes");
+ hotelsItem.innerHTML = `
+   <img class="img_hotel" src="${item.imageUrl}" alt="${item.name}">
+   <div class="text_hotel">${item.name}</div>
+   <div class="location">${item.city}, ${item.country}</div>`;
+ homesInfoPanel.append(hotelsItem);
+});
+});
+
+// записывает все, что произошло в fetch в content
+document.addEventListener('DOMContentLoaded', async function() {
+const content = await getResponse ()
+return content
+});
+
+async function getResponse() {
+const WriteContent = sessionStorage.getItem(content); // получить из памяти инфу
```

Comment:
here you use variable "content", but you should refer to key, using "", for example:
const writeContent = sessionStorage.getItem("content");